pnpm-lock.yaml

```diff
@@ -8558,6 +8570,12 @@ packages:
     resolution: {integrity: sha512-SCBrcUttktrqC5facpd29d1bbOee1vWqt+Eg2Wggjvu14A9yWbJKYjfbeHCJYiEJoaFft+WuW4Vq5RSy/QcRjw==}
     engines: {node: '>=18.12.0', npm: '>=8.19.2'}
 
+  '@wordpress/notices@5.17.0':
```

Comment:
It'd be nice if you could match the versions with the rest of the monorepo to avoid us having both `@wordpress/notices@5.17.0` and `@wordpress/notices@5.19.0` in use.

(although if you don't, we'll wind up having a Renovate PR to do it at some point 🤷)

Reply:
Of course. Before merging I'll check the versions of my added packages, because this one got bumped 3 days ago in trunk and missed it.